package/harvester-os/iso/boot/grub2/grub.cfg

```diff
@@ -0,0 +1,35 @@
+search --file --set=root /boot/kernel.xz
+set default=0
+set timeout=10
+set timeout_style=menu
+set linux=linux
+set initrd=initrd
+
+source (${root})/boot/grub2/harvester.cfg
+
+if [ "${grub_cpu}" = "x86_64" -o "${grub_cpu}" = "i386" ];then
+    if [ "${grub_platform}" = "efi" ]; then
+        set linux=linuxefi
+        set initrd=initrdefi
+    fi
+fi
+if [ "${grub_platform}" = "efi" ]; then
+    echo "Please press 't' to show the boot menu on this console"
+fi
+set font=($root)/boot/x86_64/loader/grub2/fonts/unicode.pf2
+if [ -f ${font} ];then
+    loadfont ${font}
+fi
+menuentry "Harvester Live ${harvester_version}" --class os --unrestricted {
```

Comment:
Should we print “Harvester Installer” given that it’s not a live OS for users

Reply:
Make sense. Updated.